packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -488,6 +489,31 @@ export interface RepositoryProps {
    * @default - No description.
    */
   readonly description?: string;
+
+  /**
+   * The asset the repository with the given branch should be initialized
```

Comment:
```suggestion
   * Information about code to be committed to a repository after it is created.
   * This will only take effect at creation time. Once the repository has been
   * initialized, any changes to this property will be ignored.
```